deps/build.jl

```diff
@@ -14,9 +14,10 @@ const DATA_FOLDER = Pkg.dir("PowerSystems/data")
 function download_data()
 
     if !isdir(DATA_FOLDER)
-        mkdir(DATA_FOLDER)
-        temp_folder = mktempdir()
-        temp_data_zip = joinpath(temp_folder, "data.zip")
+        mkpath(DATA_FOLDER)
```

Comment:
This is making the Windows check fail, why did you change it back? Please remove this commit.

Reply:
I didn't make that change